.ci/check_unsupported.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+# Non-zero exit code is what we want to check
+set +e
+
+# Enable capturing the non-zero exit status of coverage instead of tee
+set -o pipefail
+
+sed -i.bak -E '/^(mypy-lang|guess-language-spirit)/d' bear-requirements.txt
+
+coverage run setup.py install | tee setup.log
```

Comment:
`coverage run`?

Reply:
Used to see what code was touched by the py27 & py34 code paths.

Reply:
I've done a build removing this.  it should be ok.